ra2ce/analysis/losses/resilience_curves/README.md

```diff
@@ -0,0 +1,4 @@
+# ResilienceCurves
+
+In this subpackage all logic is found regarding reading and using resilience curves.
+The resilience curves are read from a CSV defined in `AnalysisConfigData.AnalysisSectionLosses.resilience_curves_file`.
```

Comment:
Refer to the dataclass, not the file containing it. 
```suggestion
The resilience curves are read from a CSV defined in the `ResilienceCurvesFile` dataclass (`AnalysisConfigData.AnalysisSectionLosses.resilience_curves_file`).
```

Reply:
The reference is to the config setting. The dataclass doesn't contain the reference to the CSV.